model_path: "${HOME}/RL/部署所需模型和文件/SAC_actor.onnx"
min_obs_dim: 20  # obs_min 的维度（20）
history_length: 50
obs_dim: 25
max_linear_speed: 0.5
max_angular_speed: 1.0
min_obs_distance: 0.2
rays_per_sector: 8  # 每个扇区内部的采样射线数（用于更稳健地估计扇区最小距离）
sparse_path_distance: 2.5  # 路径稀疏化距离（米），每sparse_path_distance米保留一个路径点
debug: false  # 是否开启调试模式
output_observations_file: "${HOME}/nav2_rl_controller/observations.txt"  # 观测数据输出文件路径
output_img_file: "${HOME}/nav2_rl_controller/observation_image.png"  # 障碍地图图像输出文件路径
output_compute_file: "${HOME}/nav2_rl_controller/compute_times.txt"  # 计算过程输出文件路径
